swagger.yaml

```diff
@@ -0,0 +1,249 @@
+swagger: "2.0"
+
+info:
+  title: "Tweet API"
+  description: "This provides an API for creating a message (post), reading a message (get), editing a message (put), delete a message (delete)."
+  version: "1.0"
+
+host: localhost
+
+tags:
+- name: "tweets"
+
+schemes:
+  - "https"
+
+consumes:
+  - application/json
+produces:
+  - application/json
+
+paths:
+
+  /tweets:
+  
+    get:
+      tags:
+      - tweets
+      description: "search for a user's tweets"
+      produces:
+      - application/json
+
+      parameters:
+      - in: "query"
+        name: userName
+        required: false
+        type: string
+
+      - in: "query"
+        name: tweetId
+        required: false
+        type: integer
+
+      responses:
+        200:
+          description: Successful response
+          schema:
+            $ref: "#/definitions/tweetsData"
+        400:
+          description: The specified user ID is invalid
+          schema:
+            $ref: "#/definitions/Error"
+        404:
+          description: page not found
+          schema:
+            $ref: "#/definitions/Error"
+        405:
+          description: Invalid input
+          schema:
+            $ref: "#/definitions/Error"
+        default:
+          description: Unexpected error
+          schema:
+            $ref: "#/definitions/Error"
+
+    post:
+      tags:
+      - tweets
+      description: "post a new tweet"
+      produces:
+      - application/json
+
+      parameters:
+        - name: body
+          in: body
+          description: "tweet will be created and posted"
+          required: true
+          schema:
+            $ref: "#/definitions/tweetsData"
+
+      responses:
+        200:
```

Comment:
Instead of returning a `200` for POST method, return `201` is more accurate. For example: https://github.com/osu-mist/slay-the-spire-cards-api/blob/master/swagger.yaml#L144-L147